codebase-manager.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * Plugin Name: VIP Codebase Manager
+ * Description: Tools for managing the codebase (plugins, themes, etc) within the VIP ecosystem.
+ * Version: 1.0.0
+ * Author: Automattic
+ */
+
+if ( is_admin() && file_exists( __DIR__ . '/codebase-manager/codebase-manager.php' ) ) {
```

Comment:
Under what uncontrolled condition might the file not exist? Is this file_exists() check redundant?